### PR TITLE
Add tox packaging helpers

### PR DESCRIPTION
**Pull request by ashwoods**
As mentioned on #473, this pull requests add some tox helpers for release management. 
As the setup.py uses versioning from scm, just create a tag for correct versioning.

This is for manual signed releases. An alternative would be to use travis deploy.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -46,3 +46,39 @@ commands =
...
+basepython = python3.6
+skip_install = true
+deps =
+    sphinx
+    doc8
+commands =
+    doc8 docs/
+
+[testenv:readme]
+basepython = python3.5
```
Comment (blueyed) [inline tox.ini]: Why only 3.5 here (and below)?
Could also be 3.6?!

Comment (ashwoods) [reply tox.ini]: Twine 1.9.1 doesn't officially support 3.6, as well as some of the other tools. I set all the max versions with which I have worked with.

**Comment by ashwoods** [comment]
@blueyed will do tonight!

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**